pkg/api/v1/zz_generated.conversion.go

```diff
@@ -974,6 +980,40 @@ func Convert_api_ContainerStatus_To_v1_ContainerStatus(in *api.ContainerStatus,
 	return autoConvert_api_ContainerStatus_To_v1_ContainerStatus(in, out, s)
 }
 
+func autoConvert_v1_ConversionError_To_api_ConversionError(in *ConversionError, out *api.ConversionError, s conversion.Scope) error {
```

Comment:
Hmm, not sure how this change slip in. I guess the conversion generator generates for all types in pkg/api/v1.